CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ## Deprecated
 - Keyword `replicates` of `run!` is deprecated in favor of `ensemblerun!` and will be invalid in a future version.
 - `paramscan` with `replicates` is deprecated and will be invalid in a future version.
+- All the scheduler names have been deprecated in favor of the same schedulers but prefixed with `schedule_`.
```

Comment:
Here I think we should explicitly state all name changes.

Reply:
Added.